src/controller/java/zap-generated/chip/devicecontroller/ClusterInfoMapping.java

```diff
@@ -697,8 +697,8 @@ public void onError(Exception error) {
     }
   }
 
-  public static class DelegatedScenesClusterCurrentGroupAttributeCallback
-      implements ChipClusters.ScenesCluster.CurrentGroupAttributeCallback,
+  public static class DelegatedScenesClusterLastConfiguredByAttributeCallback
```

Comment:
Where did  DelegatedScenesClusterCurrentGroupAttributeCallback go ?

Asking because this was causing me problems before (it is a group_id item ... which I had to special-case)

Reply:
I think this might be related to the exceptions that are under ChipTypesHelper.js#asBasicType
I solved this for vendor_id but I may have to add group id to the exception list as well